source/TradingPanel.cpp

```diff
@@ -215,7 +215,7 @@ void TradingPanel::Draw()
 	tradeUi->Draw(info, this);
 }
 
-
+#include <iostream>
```

Comment:
```suggestion

```
I'll pretend I didnt see this x)
dont worry I use it to test too just dont forget to remove it especially in the middle of nowhere

Reply:
(faceplam)